src/Mesh/babylon.mesh.ts

```diff
@@ -1061,6 +1061,56 @@
             this._geometry.toLeftHanded();
             return this;
         }
+        
+        /**

```

Comment:
Hello not sure this is for the framework. I would more see it in the snippets in the Doc (gamelets or utilities)

Reply:
Aw, did I at least do my first TS contribution correctly?

Not to pressed if it gets in or not, just wanted to see if I am matching the conventions correctly or not.

Where are the gamelets or utilities, so I can look up a reference for presenting this info.